src/routes/cache-hooks/entities/new-message-confirmation.entity.ts

```diff
@@ -0,0 +1,6 @@
+import { EventPayload, EventType } from './event-payload.entity';
+
+export interface NewMessageConfirmation
```

Comment:
I chose this `NewMessageConfirmation` name because both `MessageConfirmation` and `NewConfirmation` are already used in the CGW. Instead of using `New*` I also thought of a `*Event` suffix, but it would be the only event using that suffix, so I decided on `NewMessageConfirmation`. 

If you have a better alternative here, suggestions are welcome. 🙂

Reply:
This is ok from my side! 👍

We can consider `Event` as a suffix. The location of the class should also help when importing it (even when we have duplicate names) but I agree that it's easier to make the mistake of importing the wrong type if we use the same names.

Reply:
Yeah, no strong opinion on this. We could think of adding `*Event` suffix to all webhook events in a separate issue, especially if the list keeps growing.